models/cartIntegration/fileio.go

```diff
@@ -89,9 +89,9 @@ func UploadFile(file multipart.File, api_key string) error {
 			return err
 		}
 
-		custPartNum := cp.integrationExists(integrationLookup)
+		custPartNum, integExists := cp.integrationExists(integrationLookup)
 		if custPartNum != cp.CustomerPartID {
-			if custPartNum > 0 {
+			if integExists {
```

Comment:
I don't see how this logic changes anyting. now we're passing false when before we were passing 0. What are some example values, where the code you changed makes something different occur?